src/version.h

```diff
@@ -1,5 +0,0 @@
-
-#ifndef RDM_VERSION

```

Comment:
This is unrelated change, which breaks compilation of RDM. Please, remove this change. Thanks!